src/scheduler.h

```diff
@@ -35,6 +36,8 @@ class CScheduler
     CScheduler();
     ~CScheduler();
 
+    std::thread m_service_queue;
```

Comment:
I think `m_service_queue` is a bit of a weird name for a data structure that's not a queue. `m_service_thread` maybe?

Reply:
Yea. I've changed this to `m_service_thread`.